applications/nublado/values.yaml

```diff
@@ -675,6 +675,60 @@ secrets:
   # user's lab.
   templateSecrets: true
 
+# If we're installing tutorials at a given site, this cronjob controls
+# their refresh.
+cronjob:
+  # -- Enable tutorials refresh?
+  enabled: false
+
+  # -- Source for repository to clone
+  gitSource: "https://github.com/lsst/tutorial-notebooks"
+
+  # -- Target where repository should land
+  gitTarget: "/project/cst_repos/tutorial-notebooks"
+
+  # -- Branch of repository to clone
+  gitBranch: "main"
+
+  # -- Where repository volume should be mounted
+  targetVolumePath: "/project"
+
+  # -- Repository volume definition
+  # @default -- See `values.yaml`
+  targetVolume:
+    mountPath: "/project"   # Conventional
+    server: "10.234.16.4"   # NetApp always gets this IP, apparently
+    path: "/project-share"  # Assigned by Terraform
+
+  image:
+    # -- Repository cloner image to use
+    repository: "ghcr.io/lsst-sqre/repo-cloner"
+
+    # -- Pull policy for the controller image
+    pullPolicy: "IfNotPresent"
+
+    # -- Tag of Nublado controller image to use
+    # @default -- The appVersion of the chart
+    tag: null
+
+  # -- Resource limits and requests for the tutorials cronjob
+  # @default -- See `values.yaml`
+  resources:
+    limits:
+      cpu: "1"
+      memory: "1Gi"
+    requests:
+      cpu: "50m"
+      memory: "50Mi"
+
+  # -- Tolerations for the tutorials cronjob.
+  tolerations: []
+  # -- Affinity rules for the tutorials cronjob.
+  affinity: {}
+  # -- Schedule for the tutorials cronjob.
+  schedule: "43 * * * *"
```

Comment:
I'm wondering if this is excessive, and if once or twice a day would be sufficient?

Reply:
I'd like to see CST updates quickly, and one pull from a repo per hour per site is...not that excessive.  I think an hour is a reasonable default, but of course I do since that's why I chose the value.